lib/TranquilForestLandingPage.dart

```diff
@@ -62,16 +62,24 @@ class TranquilForestLandingPage extends StatelessWidget {
 
               // Activity for Goal Setting
               activityCard('Twilight Alley', context, Icons.flag,
-                  'Journal some of your thoughts',
+                  'Journal some of your thoughts and track your mood.',
                   builder: (context) => TwilightAlleyIntro(),
                   // colors
                   cardColor: Colors.white.withOpacity(0.9), shadowColor: Colors.black.withOpacity(0.3), iconBackgroundColor: Colors.teal[700],iconColor:  Colors.white, textColor: Colors.teal[700], subTextColor: Colors.teal[600]
               ),
 
+              // Activity for calming visualization
+              activityCard('Calming Cliff', context, Icons.landscape,
+                  'Calm yourself and realize that there is so much out there.',
+                  builder: (context) => CalmingCliffsIntro(),
```

Comment:
You're missing the import for this function

Reply:
https://github.com/Kenneth-W-Chen/mindfulness-app/pull/26/commits/a5cd68340579aa0d82febee6ef93fababcedda50